README.md

```diff
@@ -0,0 +1,97 @@
+# Music Library API
+
+The Music Library API is a powerful tool for managing and organizing your music collection. It provides a set of endpoints that allow you to perform various operations such as adding, updating, deleting and retrieving music information.
+
+## Features
+
+- Create artists and albums.
+- Retrieve information about artists and albums.
+- Update artists and albums.
+- Delete existing artist and album details.
+
+## Installation
+
+1. Clone the repository:
+
+`git clone https://github.com/dsbahia/music-library.git`
+
+2. Install the dependencies:
+
+`npm install`
+
+3. Start the PostgreSQL database using Docker:
+
+`docker run --name music-library-db -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres`
+
+4. Store your environment variables:
+
+- Check the .env.example file to see what should be included.
```

Comment:
There is no `.env.example` in the project, you could add one or just change this line to show what the env variables are named